bitnami/redis/templates/sentinel/statefulset.yaml

```diff
@@ -16,6 +16,10 @@ metadata:
   {{- $annotations := include "common.tplvalues.merge" ( dict "values" ( list .Values.sentinel.annotations .Values.commonAnnotations ) "context" . ) }}
   annotations: {{- include "common.tplvalues.render" ( dict "value" $annotations "context" $) | nindent 4 }}
   {{- end }}
+  {{- if or .Values.commonLabels .Values.sentinel.labels }}
+  {{- $annotations := include "common.tplvalues.merge" ( dict "values" ( list .Values.sentinel.labels .Values.commonLabels ) "context" . ) }}
+  labels: {{- include "common.tplvalues.render" ( dict "value" $labels "context" $) | nindent 4 }}
+  {{- end }}
```

Comment:
Could you please add this in a separate diff bumping Redis version?

Reply:
I have removed this chunk, I will open a dedicated PR for redis.